dc-model-jackson/src/test/resources/serializedTestObjects/identifiable/Identifier.json

```diff
@@ -1,4 +1,5 @@
 {
+  "objectType" : "identifier",
```

Comment:
Vielleicht auch besser in Großbuchstaben?

Reply:
done